demo/date-picker-basic-demos.html

```diff
@@ -47,9 +47,10 @@ <h3>Date range</h3>
     <p>Use two date pickers to create a date range selector.</p>
```

Comment:
Let's update the description to mention the component used in demo:

```html
Wrap two date pickers in <a href="https://vaadin.com/components/vaadin-custom-field"><code>&lt;vaadin-custom-field&gt;</code></a> to create a date range selector.
```

Reply:
Done